### PR TITLE
defining lv_group_remove_all_objs

### PR DESCRIPTION
**Pull request by ali-rostami**
`lv_group_remove_all_objs` is a function for removing all objects from a group and making it free of objects.

### CODE REVIEW

File: src/lv_core/lv_group.h [hunk 1]
```diff
@@ -113,6 +113,12 @@ void lv_group_add_obj(lv_group_t * group, lv_obj_t * obj);
  */
 void lv_group_remove_obj(lv_obj_t * obj);
 
+/**
+ * remove all objects from a group
```
Comment (kisvegabor) [inline src/lv_core/lv_group.h]:
```suggestion
 * Remove all objects from a group
```

File: src/lv_core/lv_group.c [hunk 2]
```diff
@@ -193,6 +193,28 @@ void lv_group_remove_obj(lv_obj_t * obj)
     }
 }
 
+/**
+ * remove all objects from a group
```
Comment (kisvegabor) [inline src/lv_core/lv_group.c]:
```suggestion
 * Remove all objects from a group
```

**Review by kisvegabor — COMMENTED** [review]
Minor formatting

**Comment by kisvegabor** [comment]
Looks good, thank you!

I've tried the suggestion feature of GitHub in PRs to format the comments. It seems I can't commit from here because I have no permission. @ali-rostami can you commit the suggestions?

**Comment by ali-rostami** [comment]
I think It's done.

**Comment by kisvegabor** [comment]
Merged, thank you!

**Pull request merged**